src/pages/Index.vue

```diff
@@ -571,7 +584,7 @@
 		flex-direction   : column;
 
 		.hero {
-			height           : 60vh;
+			height 			 : auto;
```

Comment:
Two things here:
- Align the colons
- We still need the `60vh` when the viewport is large, the goal being that I only want the `.hero` and `.keyFeatures` elements to be shown by default.
When the screen is small, we can use an `auto` property.